src/commands/requestDemon.ts

```diff
@@ -55,10 +56,18 @@ const RequestDemonCommand = {
         formattedSkillList.sort((a, b) => {
           return a.level - b.level;
         });
+        const { AttachmentBuilder, EmbedBuilder } = require('discord.js');
```

Comment:
this `AttachmentBuilder` import should be moved to the main discord.js import at the top of the file, where EmbedBuilder is already imported from